force-app/main/default/classes/TestPopulateOpportunity.cls

```diff
@@ -0,0 +1,24 @@
+@isTest(seealldata=false)
+public class TestPopulateOpportunity {
+    @isTest static void testPositive(){
+        Opportunity opp = new Opportunity(Name='Opportuity',CloseDate=Date.newInstance(2019,11,05),StageName = 'Prospecting');
+        insert opp;
+        Opportunity opp1 = [SELECT StageName FROM Opportunity WHERE id=:opp.Id];
```

Comment:
You do not need to query again the inserted record in case of updating a field value and calling an update on it.
Following is also work the same:
```
opp.StageName = 'Closed Won'
update opp;
```